src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/EditCyclePage.tsx

```diff
@@ -135,7 +135,8 @@ export function EditCyclePage() {
           <Button
             type="primary"
             onClick={() => setConfirmModalOpen(true)}
-            disabled={!formHasUpdated || Boolean(error)}
+            // Removed the disabled state for now, as it was causing issues.
```

Comment:
What was the issue here sir?

Reply:
its just not reliable for this and causes problems.

For instance, if you have advanced closed the button shows up as enabled, then you open advanced and its now disabled.

I think we just remove it

Reply:
Ok no worries, I'm down. Done